jctools-core/src/main/java/org/jctools/queues/SpscArrayQueue.java

```diff
@@ -175,11 +175,12 @@ public E peek() {
     }
 
     @Override
-    public int size() {
+    public final int size() {
```

Comment:
Why final? I think the size/isEmpty methods can be pulled up to CircularArrayBlahBlah, perhaps by making the lvConsumer/ProducerIndex methods protected. This refactoring will apply to all subclasses.